contracts/modules/CompoundManager.sol

```diff
@@ -358,9 +360,13 @@ contract CompoundManager is OnlyOwnerModule {
      * @param _amount The amount of cToken to redeem.
      */
     function redeem(address _wallet, address _cToken, uint256 _amount) internal {
-        require(_cToken != address(0), "Compound: No market for target token");
-        require(_amount > 0, "Compound: amount cannot be 0");
+        // The following commented `require()` is not necessary as `ICToken(cToken).balanceOf(_wallet)` in `removeInvestment()` would have reverted if `_cToken == address(0)`
+        // It is however left as a comment as a reminder to include it if `removeInvestment()` is changed to use amounts instead of fractions.
+        // require(_cToken != address(0), "CM: No market for target token");
```

Comment:
I think I would prefer to keep the require. Otherwise we are making the assumption that `removeInvestment` will not change and that no new method will be added that calls `redeem`. Also, are we sure that `ICToken(cToken).balanceOf(_wallet)` reverts if `cToken = address(0)`. If I remember correctly calling a method on a EOA just returns `0x`.

Anyway, I think the code would be cleaner and more robust with this simple check.

Reply:
Yes I verified that `ICToken(cToken).balanceOf(_wallet)` reverts if `cToken` is an EOA. I think you are thinking of low level calls that succeed even when the target is an EOA (`cToken.call(abi.encodeWithSignature("balanceOf"))` would succeed and return `0x` for an EOA).

I removed that `require` because I realised that there was no way to hit it in coverage. I don't think this `require` is so critical that we want to reduce our coverage to include it (it's just there to save a bit of gas if the inputs are incorrect) so if you really want to keep it I'd suggest to move it to before the `balanceOf` call in `removeInvestment` so that it can be checked in our tests/coverage.

Reply:
If it was security related I would have wanted to keep it there, but since it's only purpose is to save some gas I'm happy to have it commented out.